pkg/monitor/monitor/util.go

```diff
@@ -478,6 +478,11 @@ func getMonitorPrometheusContainer(monitor *v1alpha1.TidbMonitor, tc *v1alpha1.T
 				MountPath: "/prometheus-rules",
 				ReadOnly:  false,
 			},
+			{
```

Comment:
What if TLS is disabled for all TiDB clusters?

Reply:
I think we can take the `tls-assets` secret as default secret.When user update monitoring cluster targets, tls-assets can hot-update.

Reply:
If the user adds one TLS tc to some non-TLS tcs, then the monitor Pod will be rolling updated automatically, I think it's OK.